tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -0,0 +1,136 @@
+                                              Direction<2>::lower_eta());
+  CHECK(map_upper_xi(lower_right_corner)[0] ==
+        local_approx(random_outer_radius_upper_xi / sqrt(2)));
+  CHECK(map_upper_eta(lower_right_corner)[1] ==
+        local_approx(random_outer_radius_upper_eta / sqrt(2)));
+  CHECK(map_lower_xi(upper_right_corner)[0] ==
+        local_approx(-random_outer_radius_lower_xi / sqrt(2)));
+  CHECK(map_lower_eta(upper_right_corner)[1] ==
+        local_approx(-random_outer_radius_lower_eta / sqrt(2)));
+
+  // Check that random points on the edges of the reference square map to the
+  // correct edges of the wedge.
+  const std::array<double, 2> random_right_edge{{1.0, real_dis(gen)}};
+  const std::array<double, 2> random_left_edge{{-1.0, real_dis(gen)}};
+
+  CHECK(magnitude(map_upper_xi(random_right_edge)) ==
+        local_approx(random_outer_radius_upper_xi));
+  CHECK(map_upper_xi(random_left_edge)[0] ==
+        local_approx(random_inner_radius_upper_xi / sqrt(2)));
+  CHECK(magnitude(map_upper_eta(random_right_edge)) ==
+        local_approx(random_outer_radius_upper_eta));
+  CHECK(map_upper_eta(random_left_edge)[1] ==
+        local_approx(random_inner_radius_upper_eta / sqrt(2)));
+  CHECK(magnitude(map_lower_xi(random_right_edge)) ==
+        local_approx(random_outer_radius_lower_xi));
+  CHECK(map_lower_xi(random_left_edge)[0] ==
+        local_approx(-random_inner_radius_lower_xi / sqrt(2)));
+  CHECK(magnitude(map_lower_eta(random_right_edge)) ==
+        local_approx(random_outer_radius_lower_eta));
+  CHECK(map_lower_eta(random_left_edge)[1] ==
+        local_approx(-random_inner_radius_lower_eta / sqrt(2)));
+
+  // Test forward Jacobian numerically.
+  const double dxi = 1e-5;
+  const double deta = 1e-5;
+  const double xi = real_dis(gen);
+  CAPTURE_PRECISE(xi);
+  const double eta = real_dis(gen);
+  CAPTURE_PRECISE(eta);
+  const double inner_radius = inner_dis(gen);
+  CAPTURE_PRECISE(inner_radius);
+  const double outer_radius = outer_dis(gen);
+  CAPTURE_PRECISE(outer_radius);
+  const int direction_number = int_dis(gen);
+  CAPTURE_PRECISE(direction_number);
+  const Direction<2> direction{
+      static_cast<size_t>(direction_number > 1),
+      direction_number % 2 == 1 ? Side::Upper : Side::Lower};
+  const CoordinateMaps::Wedge2D map{inner_radius, outer_radius, direction};
+  const std::array<double, 2> test_point{{xi, eta}};
+
+  const auto numerical_d_dxi = numerical_derivative(map, test_point, 0, dxi);
+  const auto numerical_d_deta = numerical_derivative(map, test_point, 1, dxi);
+  const auto jacobian = map.jacobian(test_point);
+
+  CHECK(jacobian.get(0, 0) == local_approx(gsl::at(numerical_d_dxi, 0)));
+  CHECK(jacobian.get(1, 0) == local_approx(gsl::at(numerical_d_dxi, 1)));
+  CHECK(jacobian.get(0, 1) == local_approx(gsl::at(numerical_d_deta, 0)));
+  CHECK(jacobian.get(1, 1) == local_approx(gsl::at(numerical_d_deta, 1)));
+
+  const auto inv_jacobian = map.inv_jacobian(test_point);
```

Comment:
Extract the J J^-1 portion into a function that can then be used for testing all maps (TestMapHelpers?)